modules/dashboards/application/controllers/DashletsController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Icinga\Module\Dashboards\Controllers;
+
+use GuzzleHttp\Psr7\ServerRequest;
+use Icinga\Module\Dashboards\Form\DashletForm;
+use Icinga\Module\Dashboards\Web\Controller;
+
+class DashletsController extends Controller
+{
+    public function newAction()
+    {
+        $this->setTitle('New Dashlet');
+
+        $dashletForm = new DashletForm();
```

Comment:
I'd like to see a newline before `addContent` and maybe a better grouping of the code, e.g.

```php
        $dashletForm = (new DashletForm())
            ->on(DashletForm::ON_SUCCESS, function () {
                $this->redirectNow('dashboards');
            })
            ->handleRequest(ServerRequest::fromGlobals());

        $this->addContent($dashletForm);
```